rcl/test/rcl_add_custom_gtest.cmake

```diff
@@ -67,9 +67,14 @@ macro(rcl_add_custom_gtest target)
   if(_ARG_APPEND_LIBRARY_DIRS)
     set(_ARG_APPEND_LIBRARY_DIRS "APPEND_LIBRARY_DIRS" ${_ARG_APPEND_LIBRARY_DIRS})
   endif()
+  if(_ARG_SKIP_TEST)
+    set(_ARG_SKIP_TEST "SKIP_TEST")
+  else()
+    set(_ARG_SKIP_TEST "")
```

Comment:
Hm, I see. Looks like `cmake_parse_arguments` is actually setting it to either `TRUE` or `FALSE`. I thought that was different some (probably long) time ago. Sounds good as it is then. Sorry for the noise.